migration-engine/macros/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+extern crate proc_macro;
+
+use darling::FromMeta;
+use proc_macro::TokenStream;
+use proc_macro2::Span;
+use quote::quote;
+use syn::{parse_macro_input, AttributeArgs, Ident, ItemFn};
+
+#[derive(Debug, FromMeta)]
+struct TestOneConnectorArgs {
+    /// The name of the connector to test.
+    #[darling(default)]
+    connector: String,
+}
+
+const CONNECTOR_NAMES: &[&'static str] = &["postgres", "mysql", "sqlite"];
+
+#[derive(Debug, FromMeta)]
+struct TestEachConnectorArgs {
+    /// Comma-separated list of connectors to exclude.
+    #[darling(default)]
+    ignore: Option<String>,
+}
```

Comment:
We don't use the arguments to the attribute macro yet, but we can already parse them.